packages/components/src/custom-select-control-v2/index.tsx

```diff
@@ -1,5 +1,9 @@
 /**
  * Internal dependencies
  */
-export { default as CustomSelect } from './default-component';
-export { default as CustomSelectItem } from './custom-select-item';
+import CustomSelect from './default-component';
```

Comment:
Should we also use `CustomSelectControlV2` here for consistency?

Reply:
Addressed in 7b678d4097ffff5824d6c8b2bf78205ed8ac64fe